lib/synapse/haproxy.rb

```diff
@@ -8,6 +8,10 @@ class Haproxy
     include Logging
     attr_reader :opts
 
+    HAPROXY_CMD_BATCH_SIZE = 4
+
+    HAPROXY_BACKEND_STATE_UP = 'UP'
```

Comment:
eh i kind of changed my mind about this. we hardcode a bunch of magic strings all through the code here, i don't think extracting this one constant improves the situation.